Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -107,7 +108,17 @@ function Setup-Prerequisites
         }
         else
         {
-            Write-Host "[$($_)] module is present." -ForegroundColor $([Constants]::MessageType.Update)
+            if($_ -eq "Az.Sql" -and $AzSqlVersion -lt "4.2.0")
```

Comment:
Just need a confirmation, the string comparisons here would work as expected right? For version -lt comparisons.

Reply:
Yes, it is working fine, tested it multiple times.